spec/commands/v2/import_spec.rb

```diff
@@ -0,0 +1,93 @@
+require "rails_helper"
+
+RSpec.describe Commands::V2::Import do
+  describe "#call" do
+    let(:content_id) { SecureRandom.uuid }
+
+    let(:content_item) do
+      {
+        document_type: "foo1",
+        schema_name: "generic",
+        publishing_app: "foo",
+        title: "foo",
+        rendering_app: "foo",
+        base_path: "/bar",
+        routes: [{ "path": "/bar", "type": "exact" }],
+        details: {},
+        locale: "en",
+        state: "superseded",
+      }
+    end
+
+    let(:payload) do
+      {
+        content_id: content_id,
+        content_items: [
+          {
+            action: "PutContent",
+            payload: content_item,
```

Comment:
This usage here doesn't really make sense as we don't use the "PutContent" action for this and if we were using it we couldn't use it for "superseded"